en/core-libraries/toc-utilities.rst

```diff
@@ -1,22 +0,0 @@
-Utilities
-#########
```

Comment:
I guess it's not good for SEO to be able to access to the same page with 2 urls?
I continue to look for a solution anyway ....